src/lib/simulation/tempoross.ts

```diff
@@ -0,0 +1,137 @@
+import { Bank } from 'oldschooljs';
+import LootTable from 'oldschooljs/dist/structures/LootTable';
+
+const replaceItems = ['Fish barrel', 'Tackle box', 'Big harpoonfish', 'Tiny tempor'];
+
+const PoolCasketTable = new LootTable()
+	.add('Rune med helm', 1, 9)
+	.add('Rune full helm', 1, 9)
+	.add('Rune chainbody', 1, 9)
+	.add('Rune platebody', [2, 3], 1)
+	.add('Rune platelegs', [2, 3], 1)
+	.add('Rune plateskirt', [2, 3], 1)
+	.add('Rune sq shield', [3, 5], 1)
+	.add('Rune kiteshield', [3, 5], 1)
+	.add('Sapphire ring', [10, 14], 9)
+	.add('Emerald ring', [7, 11], 9)
+	.add('Ruby ring', [6, 8], 9)
+	.add('Diamond ring', [3, 4], 9)
+	.add('Sapphire necklace', [10, 14], 9)
+	.add('Emerald necklace', [7, 11], 9)
+	.add('Ruby necklace', [6, 8], 9)
+	.add('Diamond necklace', [3, 4], 9)
+	.add('Sapphire bracelet', [10, 14], 9)
+	.add('Emerald bracelet', [7, 11], 9)
+	.add('Ruby bracelet', [6, 8], 9)
+	.add('Diamond bracelet', [3, 4], 9)
+	.add('Sapphire ring', [20, 28], 1)
+	.add('Emerald ring', [14, 22], 1)
+	.add('Ruby ring', [12, 16], 1)
+	.add('Diamond ring', [6, 8], 1)
+	.add('Sapphire necklace', [20, 28], 1)
+	.add('Emerald necklace', [14, 22], 1)
+	.add('Ruby necklace', [12, 16], 1)
+	.add('Diamond necklace', [6, 8], 1)
+	.add('Sapphire bracelet', [20, 28], 1)
+	.add('Emerald bracelet', [14, 22], 1)
+	.add('Ruby bracelet', [12, 16], 1)
+	.add('Diamond bracelet', [3, 8], 1)
+	.add('Silver bar', [50, 100], 9)
+	.add('Gold bar', [50, 100], 9)
+	.add('Uncut opal', [25, 50], 9)
+	.add('Uncut jade', [25, 50], 9)
+	.add('Uncut red topaz', [10, 15], 9)
+	.add('Uncut sapphire', [20, 30], 1)
+	.add('Uncut emerald', [15, 25], 1)
+	.add('Uncut ruby', [10, 20], 1)
+	.add('Uncut diamond', [8, 13], 1)
+	.add('Uncut dragonstone', [2, 3], 1)
+	.add('Coins', [4000, 6000], 9)
+	.add('Loop half of key', 9, 1)
+	.add('Tooth half of key', 9, 1)
+	.add('Coins', [25000, 50000], 1)
```

Comment:
if a number is 5 digits long (or more), you need to use a numeric separator: `25_000`